templates/Web/Pages/Vue.MasterDetail/src/views/VueMasterDetail.vue

```diff
@@ -43,10 +43,13 @@ export default {
     return {
       masterDetailText: [
         {
-          paragraph: "",
-          title: "",
-          tabName: "",
-          id: 0
+          id: 0,
+          longDescription: "",
+          orderDate: "",
+          orderTotal: 0,
+          shipTo: "",
+          status: "",
+          title: ""
```

Comment:
Why were all these parameters not already there? Was it even working without them?

Reply:
no, it was not working, this is pre-fetched data, and the fields were miss named so it was broken and I discovered it by writing the `required: true` in props.